tests/csm/rest/test_capacity_quota.py

```diff
@@ -214,12 +210,12 @@ def test_40633(self):
         assert result, "Verification for get set user failed."
         self.log.info("Response : %s", resp)
         self.log.info("Step 3: Perform put object of max size")
-        self.csm_obj.verify_max_size(max_size)
+        self.csm_obj.verify_max_size(max_size, self.akey, self.skey)
```

Comment:
Return value verification missing

Reply:
tried asserting in a different lib function and it worked